.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+
+@guardian/editorial-tools-sshaccess 
```

Comment:
why ssh access instead of `@guardian/digital-cms`?

Reply:
Because I forgot digital-cms exists. Just did a search for `tools`